oojs.js

```diff
@@ -1,3 +1,24 @@
+class Playlist {
+  constructor(title, ){
+    this.title = title;
+    this.song = [];
+    this.favorite = 0;
+  }
+}
+addSong(song){
+  this.song.push(song);
+  console.log(this.song);
+}
+
+const playlistName = new Playlist("playlistName")
```

Comment:
:+1: although this would look different when you pass in the correct number of arguments.